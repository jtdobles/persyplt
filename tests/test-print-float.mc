void main(){
	printf(20.4);

	float a = 4.2 + 10.5;
	printf(a);
    
    printf(1.2 + 1.0);
}