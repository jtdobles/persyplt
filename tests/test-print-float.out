20.400000
14.700000
2.200000
